studio/components/interfaces/Billing/PlanSelection/Plans/Plans.Constants.ts

```diff
@@ -50,19 +49,17 @@ export const PRICING_META = {
       'No project pausing',
       'Email support',
     ],
-    scale: 'Additional fees apply for usage and storage beyond the limits above.',
-    shutdown: '',
-    preface: 'Everything in the base plan, plus',
     additional: 'Need more? Turn off your spend cap to Pay As You Grow ',
-    cta: 'Get Started',
+    scale: 'Additional fees apply for usage and storage beyond the limits above.',
   },
   [STRIPE_PRODUCT_IDS.TEAM]: {
+    new: true,
     name: 'Team',
-    href: '#',
-    from: true,
+    description: 'For collaborative teams with different permissions and access.',
```

Comment:
Should this be "For collaborative teams with different permissions and access **patterns**"? Or something similar?
Bonus points if you can think of shorter wording, too, so these line up:
<img width="816" alt="Screenshot 2023-01-12 at 11 29 59 pm" src="https://user-images.githubusercontent.com/10985857/212202995-deeec773-8f08-4fa8-a278-0e61d4257228.png">

Reply:
ahh right! Think i'll go with `Collaborate with different permissions and access patterns`